circle.yml

```diff
@@ -13,7 +13,7 @@ dependencies:
     - ~/.ghc-mod
     - ~/infer-linux64-v0.7.0
     - ~/nltk_data
-    - ~/pmd-bin-5.4.1
+    - ~/pmd-bin-5.6.1
```

Comment:
The changes to `circle.yml` shouldnt be needed any more due to https://github.com/coala/coala-bears/commit/a724ad7fa7ec734704e4bf4c8a8405c2a66b592e